DataAccessLayer/classes/SendinBlueManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Net.Mail;
+using Microsoft.Extensions.Configuration;
+using Models;
+using Scriban;
+using Serilog;
+using sib_api_v3_sdk.Api;
+using sib_api_v3_sdk.Client;
+
+namespace DataAccessLayer
+{
+    public class SendinBlueManager : IMailManager
+    {
+        private readonly IConfigurationRoot _config;
+        private readonly ILogger _logger;
+        private readonly SmtpClient _smtp;
+
+        public SendinBlueManager(IConfigurationRoot config, ILogger logger)
+        {
+            _config = config;
+            _logger = logger;
+            ApiKey = _config["MAIL_API_KEY"];
+            Configuration.Default.ApiKey.Add("api-key", ApiKey);
+            _smtp = new SmtpClient(_config["SMTP_SERVER"], int.Parse(_config["SMTP_PORT"]));
+            _smtp.EnableSsl = true;
+            _smtp.Credentials = new NetworkCredential(_config["SMTP_LOGIN"], _config["SMTP_PASSWORD"]);
+        }
+
+        public string ApiKey { get; set; }
```

Comment:
move it with its firends, ApiKey feels lonely here

Reply:
For some reason, Resharper send the public attribute just after the constructor.
Don't know why or if it's a correct behaviour or not
Fixed it to remain at the top with the private attributes